packages/@okta/vuepress-site/docs/guides/oin-universal-logout-overview/index.md

```diff
@@ -0,0 +1,118 @@
+
+## Universal Logout endpoint
+
+The actual endpoint URL is up to the discretion of the app developer building the endpoint for the app.
+
+### Endpoint authentication
+
+The request to the Universal Logout endpoint requires authentication so that your app knows the request is coming from Okta. When the app launches, Okta should be able to support your existing API authentication scheme, especially if it's one of the following:
+
+* OAuth 2.0 Bearer token, for example: `Authorization: Bearer X1234`
+* API key sent in a custom HTTP header
+
+### Logout request
+
+When a user should be logged out of the app, Okta makes a POST request to the Universal Logout endpoint. The request includes a JSON object that describes the user to be logged out.
+
+By default, the user's email address identifies them. If an app supports provisioning with Okta, then the user identifier within the app identifies them. The user identifier is sent in the format defined by [Subject Identifiers for Security Event Tokens](https://datatracker.ietf.org/doc/html/draft-ietf-secevent-subject-identifiers-18) as either an `EmailSubject` or `Opaque` identifier.
+
+Email address:
+
+```JSON
+{
+  "subject": {
+    "format": "email",
+    "email": "user@example.com"
+  }
+}
+```
+
+User identifier within the app:
+
+```JSON
+{
+  "subject": {
+    "format": "opaque",
+    "id": "d563aec52"
+  }
+}
+```
+
+### Logout response
+
+The response should indicate whether the logout request was successful, if the app is unable to log the user out, or if there's some other error. Okta ignores the response body and uses only the HTTP status code to indicate success or failure.
+
+> **Note:** See [Revocation details](#revocation-details) for more information on what the app should revoke to consider the logout flow successful.
+
+Your app should first validate the request authentication, and then return an error if the request is missing credentials or if the credentials are invalid. Indicate this error with an HTTP `401` or `403` response code:
+
+* `401 Unauthorized`: The request is missing authentication or the authentication is invalid.
+* `403 Forbidden`: The provided authorization is insufficient to perform the requested operation, for example: missing scope.
+
+If the request is malformed, or includes a subject identifier of an unrecognized type, your app can return HTTP `400`:
+
+`400 Bad Request`: The request is malformed, contains invalid or unrecognized properties, or an unrecognized subject identifier type.
+
+Your app should look up the user identified by the subject identifier and return `404` if the user isn't found:
+
+`404 Not Found`: The user indicated by the subject identifier isn't found.
+
+At this point, your app should attempt to log the user out. If it's not possible, your app can return HTTP `422`:
+
+`422 Unprocessable Content`: The app is unable to log the user out.
```

Comment:
Should we also include 429?

Reply:
can we come back to this question later after we get some initial feedback from people who would be implementing the logout API?

Reply:
Sure, I guess the same will go for `200` as I know it's common (bad practice) to return `200` when it should be a `204`. Not sure if we should be strictly validating it's `204` vs accept `200` as well